source/simulator/core.cc

```diff
@@ -989,6 +989,8 @@ namespace aspect
   template <int dim>
   void Simulator<dim>::setup_dofs ()
   {
+    signals.edit_parameters(*this, parameters);
+
```

Comment:
I wonder if this is the right place to call the signal. How about at the end of `parse_parameters()`? Or what was your reasoning to call it here?